internal/cypress/v1/config.go

```diff
@@ -264,7 +266,25 @@ func (p *Project) Validate() error {
 	return nil
 }
 
-func shardSuites(rootDir string, suites []Suite, ccy int) ([]Suite, error) {
+// excludeSauceIgnorePatterns excludes any match with sauceignore content.
+// If loading and parsing the sauceignore content fails, no filtering is applied.
+func excludeSauceIgnorePatterns(files []string, sauceignoreFile string) []string {
```

Comment:
Hm, we have this function multiple times, that is completely identical from what I can tell.
Would this function make more sense in the `sauceignore` package, so that it can be reused?

Reply:
That was my initial guess. Since we have already a lot of duplication I though that was better to keep files self-consistent.
I moved that function to the `sauceignore` package.